src/DynamoCoreWpf/UI/GuidedTour/CustomRichTextBox.cs

```diff
@@ -91,20 +93,31 @@ private static FlowDocument GetCustomDocument(string Text)
                     bBulletListActive = true;
                 }
 
-                //A format for inserting a hyperlink between text was found
-                //The hyperlink name is the next word followed by the # char and the URL value is the one followed after the = char
+                //A format for inserting a hyperlink between text was found              
                 if (word.StartsWith("#"))
                 {
-                    string fullHyperlinkText = word.Substring(1, word.Length - 1);
-                    string linkName = fullHyperlinkText.Split('=')[1];
-                    string linkURL = linkName;
-
-                    Run run3 = new Run(fullHyperlinkText.Split('=')[0]);
-                    Hyperlink link = new Hyperlink(run3);
-                    link.IsEnabled = true;
-                    link.NavigateUri = new Uri(linkURL);
-                    link.RequestNavigate += (sender, args) => Process.Start(args.Uri.ToString());
-                    para.Inlines.Add(link);
+                    bHyperlinkActive = true;
```

Comment:
For now any purpose of this bool? if not, seems we dont really need it and can just use the if condition above?

Reply:
@QilongTang 
If you see the foreach cycle is reading word by word, then when a hyperlink format is found we need to know if we are reading the Hyperlink Text or the URL or if the hyperlink has finished, that's why we are using the bHyperlinkActive flag (we will pass for the condition "if (bHyperlinkActive)" many times depending of the hyperlink length ).
Then consider the next text as an example:

_Please click here for go to the Autodesk site #Autodesk Site=https://www.autodesk.com/_

The bHyperlinkActive will be true when "#" is found and then concatenate the words "Autodesk" and "Site" for getting the Text, finally read the URL and set the bHyperlinkActive to false.

Let me know if this clarify your question or need more detail.
Thanks